optaplanner-examples/src/main/java/org/optaplanner/examples/common/business/SolutionBusiness.java

```diff
@@ -126,16 +128,30 @@ public void setImporters(AbstractSolutionImporter<Solution_>[] importers) {
         this.importers = importers;
     }
 
-    public void setExporter(AbstractSolutionExporter<Solution_> exporter) {
-        this.exporter = exporter;
+    public void setExporters(Set<AbstractSolutionExporter> exporters) {
+        this.exporters = exporters;
+    }
+
+    public void addExporter(AbstractSolutionExporter<Solution_> exporter) {
+        if (this.exporters == null) {
+            this.exporters = new HashSet<>();
+        }
+        this.exporters.add(exporter);
+    }
+
+    public Set<AbstractSolutionExporter> getExporters() {
+        if (this.exporters == null) {
+            return new HashSet<>();
+        }
+        return this.exporters;
```

Comment:
If `exporters` are never null (see previous comment), the handling of that can disappear from this class.